packages/chat-component/src/core/parser/index.ts

```diff
@@ -63,13 +64,18 @@ export async function parseStreamedMessages({
       isFollowupQuestion = true;
       followupQuestionsBuffer.push(chunkValue);
       continue;
+      // if we're already processing questions, we don't need to check for the marker again
+      // but we need to check if we reached the end of the followup questions
     } else if (chunkValue.includes('<<') && isFollowupQuestion) {
       isFollowupQuestion = true;
       continue;
+      // this updates the index, so we add each question to a different array entry
+      // to simplify styling
     } else if (chunkValue.includes('?>') || chunkValue.includes('>')) {
```

Comment:
why are we checking for `?>` && `>` and not `?>>` && `>>`?

Reply:
Because in all tests I ran, ?> and the last > were returned as part of 2 different chunks.